examples/tests/backend_driver.py

```diff
@@ -205,7 +205,7 @@
     'multiline.py',
     'multiple_figs_demo.py',
     'nan_test.py',
-    'newscalarformatter_demo.py',
+    'plot_scalarformatter.py',
```

Comment:
Once #8069 goes through all these explicit example file references will be gone anyway, so I don't think this matters.